[WIP] Translation review

Minoru opened the pull request:
Привет!

Тоже хотел перевести игру на русский, но раз уж ты начал, то мне достаётся роль редактора. (Если ты не возвражаешь, конечно). Естественно, все правки обсуждаемы.

Пока что прочёл лишь половину. Создаю PR для того, чтобы ты знал о моей работе и не создавал пока PR в основном репозитории.

Я сегодня точно закончу первый проход, потом нужно будет сделать ещё как минимум парочку. К пятнице нужно успеть :)

Я пока что не обращаю внимания на то, как мои правки выглядят в игре. Как только сделаем качественный перевод, можно будет заняться расставлением переводов строк и прочими оформительскими задачами.

Minoru committed:
Review round 1, 48% done

Minoru committed:
Копикот → Подражатель

Minoru committed:
Review round 1, done

Minoru commented:
Так-с, первый проход завершён. Попутно [пожаловался в апстрим](https://github.com/ncase/trust/issues/17) касательно множественных форм — нам же нужно писать «1 раунд», «2 раунда», «5 раундов», а код позволяет только две формы (единичное и множественное число).

Из крупных проблем, с моей точки зрения, осталась всего одна: в русском не принято всё время «тыкать», а перевод сейчас слишком буквальный. Нужно перефразировать текст, чтобы он читался более естественно. Этим я займусь во втором раунде (скорее всего, завтра).

Из мелочи переводы всякого сленга («chillax», «jerkwad») и отсылки ([Kumbaya](https://en.wikipedia.org/wiki/Kumbaya), [Hatfield-McCoy](https://en.wikipedia.org/wiki/Hatfield–McCoy_feud)). Эти вещи лучше перевести плохо, чем не перевести никак, потому что непонятные слова сбивают с толку значительно сильней, чем просто неудачные. Этим я займусь в третьем раунде.

toby3d commented:
@Minoru, привет! Благодарю за инициирование PR!

## Про статус перевода
Мои последние коммиты как раз относятся к категории "чернового" перевода, а потому - страшно нуждаются в правках и исправлениях: опечатки, знаки препинания и форма предложений (я оставил как в оригинале с небольшими исключениями), вопросы с транслитерацией имён и так далее. Так что вы делаете всё правильно, за что большое спасибо!

## Про технику
Самая большая проблема в переводе любого медиа - насколько хорошо слова вписываются в выделенные им пространства. Идеальный вариант, когда "ванильная" версия содержит в себе текст с, как минимум, 1/3 от занимаемого текстом свободного пространства. Увы, здесь такого не наблюдается. Как вывод: "пиши - сокращай" больше в угоду вместимости, чем полного сохранения изначального смысла со всеми художественными элементами.

## Про дизайн
Менее важная, но, всё-таки, проблема - шрифты. О чём я, кстати, уже [дал знать в репозитории](https://github.com/ncase/trust/issues/10) оригинальной игры, но ответа я пока не получил. А если внедрять новые, то создавать технические неудобства автору. А также наиболее вероятно нарушать лицензию. Что делать - непонятно.

## Про стиль
По поводу первого лица - большинство проектов Ники являются по сути личным диалогом с игроком. Это можно пронаблюдать в "[Neurotic Neurons](http://ncase.me/neurons/)" или "[Coming Out Simulator 2014](http://ncase.itch.io/coming-out-simulator-2014)", например. Так что я решил сохранить традицию и сберечь тон особой близкой беседы с игроком о проблеме, описываемой в эксперименте.

## P.S.
После последних проверок я отправлю письмо о добавлении этой версии локализации непосредственно в оригинальный репозиторий игры и попрошу указать и ваше имя тоже. Примерные планы - закончить до конца этой недели (раньше - лучше).

Давайте сделаем это! :D

toby3d commented on words.html:
```diff
@@ -150,70 +150,70 @@
-Но если ты обманываешь и он доверился, то ты получаешь 3 монеты по цене 1. (счёт: +3 vs -1)
-<b>Потому: ты ещё "должен" ЖУЛЬНИЧАТЬ.</b>
+Потому что если вы оба доверяетесь, то вы оба жертвуете одной монетой, чтобы получить три. (счёт: +2 vs +2)
+Но если ты обманываешь и он доверяется, то ты получаешь 3 монеты, а он теряет одну. (счёт: +3 vs -1)
```
`он доверяется` здесь выглядит криво, во время чтения, на мой взгляд.

toby3d commented on words.html:
```diff
@@ -150,70 +150,70 @@
-<b>Потому: ты ещё "должен" ЖУЛЬНИЧАТЬ.</b>
+Потому что если вы оба доверяетесь, то вы оба жертвуете одной монетой, чтобы получить три. (счёт: +2 vs +2)
+Но если ты обманываешь и он доверяется, то ты получаешь 3 монеты, а он теряет одну. (счёт: +3 vs -1)
+<b>Потому: ты все равно "должен" ЖУЛЬНИЧАТЬ.</b>
```
Дилемма е/ё

toby3d commented on words.html:
```diff
@@ -258,8 +258,8 @@
 </p>
 
 <p id="character_tft">
-<b>КОПИКОТ:</b>
```
Большой вопрос: Copycat звучит как любопытная игра слов. В своей черновой версии я криво, но попытался сохранить принадлежность персонажа (кота) и его имя. `Подражатель` звучит понятней, но тогда теряется всё вышеперечисленное. Мяу. :cry:

toby3d commented on words.html:
```diff
@@ -335,26 +335,26 @@
 </p>
 <p id="tournament_5">
-И, по факту, еще <i>до</i> Рождества несколько фронтов <i>уже</i> установили
+Фактически еще <i>до</i> Рождества несколько фронтов <i>уже</i> установили
```
Запятая и е/ё (с последним - мой косяк).

toby3d commented on words.html:
```diff
@@ -485,59 +484,59 @@
-но как только с ними было покончено, им пришлось столкнуться с <span class="tft">Копикотами</span>:
-которые <i>также</i> хороши, но не наивны.
+но как только с ними было покончено, им пришлось столкнуться с <span class="tft">Подражателями</span>:
+которые <i>приятны</i>, но не наивны.
```
Ну... Пока под вопросом.

Minoru replied:
В оригинале «They had to face the Copycats, who *are* nice, but not naive». «Nice» означает «приятные, вежливые», «хороши» означает «ловко справляются с делами; красиво выглядят». Таким образом, переводить «nice» как «хороши» никак нельзя!

toby3d commented on words.html:
```diff
@@ -549,30 +548,31 @@
 
 <p id="evo_11">
-Взгляни вокруг. Мир полон кретинов.
+Оглянись вокруг. Мир полон козлов.
```
Опять же - под вопросом.
~~Хотя РКН констатировал, что слово "мудак" является допустимым не обсценным словом.~~

Minoru replied:
Моя логика тут в следующем: «кретин» это на тему умственных способностей, а исходное «jerkwad» явно про мудацкие склонности людей. Но писать «мудак» не хотелось, потому что это довольно сильное слово; «козлы» как-то полегче.

toby3d commented on words.html:
```diff
@@ -549,30 +548,31 @@
-<!-- Копикот является стратегией в этой повторяющейся игре доверия; она настолько сильна, что даже солдаты Первой мировой войны самостоятельно "развили" аналогичную стратегию; почему тогда мир полон ненадёжных людей? -->
-Если <span class="tft">Копикот</span> является стратегией в этой повторяющейся игре доверия, которая настолько сильна, что даже солдаты Первой мировой войны самостоятельно "развили" аналогичную стратегию, почему тогда так много ненадежных людей?
-Что вызывает нашу эпидемию недоверия?
+Раз <span class="tft">Подражатель</span> настолько удачная стратегия для этой повторяющейся игры доверия
```
`игры доверия` => `игры на доверие`?

Minoru replied:
Превосходная идея! Заменил по всему файлу.

toby3d commented on words.html:
```diff
@@ -549,30 +548,31 @@
-Если <span class="tft">Копикот</span> является стратегией в этой повторяющейся игре доверия, которая настолько сильна, что даже солдаты Первой мировой войны самостоятельно "развили" аналогичную стратегию, почему тогда так много ненадежных людей?
-Что вызывает нашу эпидемию недоверия?
+Раз <span class="tft">Подражатель</span> настолько удачная стратегия для этой повторяющейся игры доверия
+-- настолько удачная, что даже солдаты Первой мировой переизобрели её под именем "живи и дай жить" --
```
`живи и дай жить` => `живи и дай жить другим`?

toby3d commented on words.html:
```diff
@@ -549,30 +548,31 @@
 <br><br>
-Ключ в самой формулировке. "В <i>этой</i> повторяющейся игре доверия."
-До сих пор мы говорили только об изменениях в игроках:
+Ключ в самой формулировке. "Для <i>этой</i> повторяющейся игры доверия."
```
`игры на доверие`? (в моём коммите косяк с этим, сорян)

toby3d commented on words.html:
```diff
@@ -662,15 +662,15 @@
 
 <p id="noise_2">
-И, как правило, они просто вознаградили друг друга добротой и пели Kumbaya до конца времён.
+И, как правило, они просто отплатят друг другу добротой и будут до конца времён петь Kumbaya.
```
Я бы оставил `петь Kumbaya` на старом месте. На слух такой порядок слов произносится чуть легче.

Minoru replied:
«И будут петь Kumbaya до конца времён» звучит так, будто бы они пели и будут петь. Предложенная мой правка больше похожа на «займутся пением и будут продолжать до конца времён».

Вообще к этому вопросу нужно будет вернуться после замены «Kumbaya» на что-то знакомое русскоязычному читателю.

toby3d commented on words.html:
```diff
@@ -679,48 +679,48 @@
 </p>
 
 <p id="character_tf2t">
 <b>КОПИКОТЁНОК:</b><br>
```
Если `Copycat` это `Подражатель`, то как быть с `Copykitten`?

toby3d commented on words.html:
```diff
@@ -1081,7 +1088,7 @@
 
 <p id="forgot_whos_who">
-(забыл, кто есть кто? наведи на кнопки, чтобы увидеть описания каждого персонажа!)
+(забыл, кто есть кто? наведи мышь на кнопки, чтобы увидеть описания каждого персонажа!)
```
Лучше `курсор`.

toby3d commented on words.html:
```diff
@@ -1156,7 +1163,7 @@
 
 <p id="label_rules">
-устав
+правила
```
Я поменял ранее переведённое `правила` на `устав` дабы это слово влезло в ширину вкладки песочницы.

Minoru replied:
«Устав» слишком старомодно и совсем выбивается из общего разговорного стиля. Нужно подумать.

toby3d commented on words.html:
```diff
@@ -1191,13 +1198,13 @@
 </p>
 <p id="chapter_noise">
-6. Допущение ошибок
+6. Ашипки
```
Скорее всего не корректно, так как это название главы.

Minoru replied:
Не понимаю, какая разница, название это главы или просто слова посреди абзаца.

Понятное дело, что «Допущение ошибок» ближе к исходному «Making Mistaeks», чем предложенная мной правка, но по-русски так никто не пишет. Это канцелярский слог какой-то.

toby3d commented:
У-упс, кажется мне проще было начать ревью, чем спамить одиночными комментариями. Извиняюсь. :confounded:

Minoru committed:
"Игра доверия" → "игра на доверие"

Minoru committed:
Apply some review comments from @toby3d

Minoru commented:
> Самая большая проблема в переводе любого медиа - насколько хорошо слова вписываются в выделенные им пространства.

О, это любопытно! Спасибо. Я раньше только статьи переводил для своего блога, там никаких ограничений не было.

> Менее важная, но, всё-таки, проблема - шрифты.

Мне замена шрифта на аналогичный видится наилучшим решением. И автору *придётся* на это идти, если он хочет, чтобы игра была не только на английском: шрифтов, поддерживающих много языков, раз-два и обчёлся. И все «некрасивые» :)

> А если внедрять новые, то создавать технические неудобства автору. А также наиболее вероятно нарушать лицензию.

Плата за успех :) Игра под CC-0, это эквивалентно public domain. Можем делать всё, что хочем.

> По поводу первого лица - большинство проектов Ники являются по сути личным диалогом с игроком. […] Так что я решил сохранить традицию и сберечь тон особой близкой беседы с игроком о проблеме, описываемой в эксперименте.

Я жалуюсь не на первое лицо, а на постоянные «ты». Пример с потолка: «посмотри, что будет, если ты нажмёшь кнопку «Старт» vs. «посмотри, что будет, если нажать кнопку «Старт».

Теперь касательно Copycat и Copykitten. Это, конечно, круто, что автор нашёл игру слов, но, к счастью, в интерфейсе на это ничего не завязано, и мы можем спокойно заменять эти термины на всё, что угодно. Читатель ничего не потеряет, если вместо шутки автора прочтёт нашу. В худшем случае мы вместо «Copykitten» подставим что-то несмешное и избавимся от «мяу» и «мур»; в лучшем всё это будет заменено на понятную русскоязычному читателю игру слов.

toby3d commented:
> О, это любопытно! Спасибо. Я раньше только статьи переводил для своего блога, там никаких ограничений не было.

По этой причине я заменяю "словесные" числа, непосредственно, числами: `сорок` => `40`, `три монеты` => `3 монеты` и так далее. Всё для того чтобы хоть немножечко сэкономить в пространстве.

P.S.: Я могу смержить текущее состояние правок чтобы мы могли убедиться в том, влезает ли текст в установленные рамки или нет.

Minoru commented:
> P.S.: Я могу смержить текущее состояние правок чтобы мы могли убедиться в том, влезает ли текст в установленные рамки или нет.

Я у себя для этого запускаю `python3 -m http.server` и смотрю итог на http://127.0.0.1:8000 Не влезает :) Нужно поменять шрифт и заново расставить переводы строк, но я не вижу смысла этим заниматься, пока не доведу перевод до приемлемого состояния.

У тебя где-то под кнопкой «Merge» дожна быть ссылка, по которой тебе объяснят, как стянуть мою ветку к тебе в локальный клон. Что-то типа такого:

```
$ git checkout -b minoru-russian-translation-review 'master~100'
$ git pull https://github.com/Minoru/trust.git feature/russian-translation-review
```

Так ты сможешь проверять мои правки, не делая мерж.

toby3d commented:
> Нужно поменять шрифт

Уже поменял на самый похожий и со свободной лицензией.
Нет, это не Comic Sans.

Minoru commented:
Только что тоже поменял у себя на Pangolin. Местами текст уже не влазит. Потом буду чинить, сейчас главное слова подобрать.

Сегодня ничего уже не пушну, второй раунд буду заканчивать завтра. Извини!

toby3d reviewed (changes requested):
722: `привет я сначала сотрудничать`, склонение.
820: Скорее всего [косяк самого Ники](https://github.com/ncase/trust/issues/9) и это следует поправить.

toby3d commented:
> Сегодня ничего уже не пушну, второй раунд буду заканчивать завтра. Извини!

Всё 👌🏻! Укладываемся!
Снов и удачи в следующем заходе. (И спасибо за подсказки по дебагу)

Minoru commented:
> 722: привет я сначала сотрудничать, склонение.

Там в оригинале очень упрощённая речь без пунктуации и тоже с неверными 
формами: «hi i try start cooperate». Я хочу попытаться это передать. 
(Пользуясь случаем, рекомендую рассказ «Цветы для Элджернона» Дэниела 
Киза. Есть ещё роман, но он просто раздутый в десять раз рассказ.)

> 820: Скорее всего косяк самого Ники и это следует поправить.

Да, но я надеюсь, что пока мы тут подыскиваем правильные слова 
и избавляемся от лишних, Ники обновит апстрим и нам не придётся 
придумывать формулировку за него. (Хотя «anyone can win», предложенная 
в том ишшуе, мне нравится. А ещё я не перестаю улыбаться от URL: 
github.com/ncase/ **trust/issues** /10).

Minoru committed:
Review round 2, done

Minoru commented:
Во втором раунде я пытался избавиться от лишних «ты» и заставить текст читаться более естественно. Много экспериментальных правок, ревью наверняка будет горячим :)

Из крупных проблем всё ещё остаются:
* отсылки к Kumbaya и вражде двух семей;
* мне не нравится, что текст в некоторых местах явно указывает пол игрока («Ты поставил на [CHAR}»). Хотел бы от этого избавиться, чтобы не мешать девушкам погружаться в игру.

В целом текст, с моей точки зрения, уже смотрится достойно.

В третьем раунде — завтра, после применения твоих замечаний — планирую заняться втискиванием текста в свободное место.

toby3d reviewed (commented):
Делаю ревью как комментарий, так как мои замечания в этот раз вообще не обязательны к исполнению и являются исключительно субъективным взглядом:
101: Думаю, можно оставить и просто "поиграть" чтобы чуть-чуть сэкономить место и не потерять смысл.
164: может лучше `Зачем позволять клянчить попрошайке?`, т.е. переставить `клянчить` с конца предложения?
191: `доверяетесь` выглядит как-то криво на мой взгляд. Может оно есть в словарях, но как-то meh...
881: Думаю вариант `что узнали сегодня!` звучит получше.

Minoru committed:
Review round 2 feedback, done

Minoru commented:
101, 164, 881: применил, спасибо!

А по 191 предложения есть? Я не понимаю, что именно плохо в "доверяетесь", поэтому не знаю, что написать взамен.

Minoru commented:
Вместо третьего раунда уже битый час пытаюсь заставить браузер использовать Pangolin. Вот буквально позавчера работало, а сегодня уже нет!  Не работают ни твоя ветка, ни моя.

Видимо, я что-то упускаю. Завтра уже этим займусь.

toby3d commented:
@Minoru Только что обновил вашу ветку и локально запустил веб-сервер. Всё в порядке со шрифтами.

![image](https://user-images.githubusercontent.com/5412322/28893913-8ef40174-77ec-11e7-9b7f-546262e8625c.png)

Но, боюсь, придётся править ещё что-нибудь когда я обновлю свой репозиторий к состоянию оригинального.

Minoru committed:
Make Pangolin work for me

Minoru commented:
Удалось-таки заставить шрифт отображаться. Проблема была в некорректном `@font-face`: в `font-family` было указано несколько значений.

Приступаю к третьему раунду!

Minoru committed:
-- → &mdash;

With Pangolin font the "two dashes" thing became too apparent, so
I replaced it with a proper symbol.

Minoru committed:
Review round 3, done

Minoru committed:
Mention myself on the start page

Minoru commented:
Итак:

* от Kumbaya просто уклонился, написав «жили припеваючи»;
* Хэтфилдов и Маккоев оставил как есть;
* игра всё ещё обращается к игроку в мужском роде;
* сноски и peeps/index.html ещё даже не читал (ни в переводе, ни в оригинале).

Тем не менее, считаю текст готовым к первому публичному релизу. Можешь при мерже сделать squash; не думаю, что моя история изменений будет кому-то интересна.

Я до вторника буду занят и по этому переводу никакой новой работы сделать не смогу. (Замечения рассмотреть у меня время будет.)

Раз перевод хостится у тебя и можно достаточно просто выкатывать апдейты, я на следующей неделе займусь вычиткой сносок и peeps/index.html.

Ещё было бы неплохо, если бы ты в в этом репозитории включил issues (Settings → Options → Features → ☑ Issues). Тогда можно будет для отдельных проблем сделать отдельные ишшуи и рассматривать их там, не превращая ревью пулл-реквестов в обсуждение всего и вся.

^_^

toby3d commented:
@Minoru Окей. Issues включу прямо сейчас, PR - тоже склею и закрою. А вот репозиторий, пожалуй, перенесу в @NotDotTeam, так как некоторые знакомые случайно считают будто эту игру сделал я. Непорядок.

balamosh commented:
Привет, только в голову пришла идея о локализации этого проекта, а тут уже оказывается все сделано и без меня. И это круто! Но если вам вдруг понадобится помощь например с теми же сносками, то я буду рад помочь.

toby3d commented:
@balamosh Приветствую!

Да, с основным содержанием игры вроде как почти разобрались. Осталось проверить перевод заметок и страницы со спонсорами. Создайте новый PR где исправите мой черновой перевод.

Minoru commented:
@toby3d Спасибо! Касательно переноса репозитория: не забудь добавить ссылку в [свой комментарий о переводе](https://github.com/ncase/trust/issues/5#issuecomment-318541832), чтобы народу, читающему тот ишшуй, проще было найти репозиторий.

@balamosh Привет! Ты также можешь перечитать words.html и убедиться, что перевод выглядит нормально; мы с @toby3d на этот файл смотрим уже неделю и вполне можем упускать очевидные косяки.

balamosh commented:
@toby3d Из заметок пока что только бросились в глаза 2 фразы "You can't make this shit up" и "Pincky promise". Первое я бы перевел как "Такое сложно выдумать" и "Зуб даю". Хотя именно в тексте для второго лучше звучит "Честное пионерское". В общем как пройдусь еще пару раз по тексту - добавлю PR.

@Minoru Я в принципе поиграл с https://toby3d.github.io/trust/ и в целом все очень добротно! Есть конечно моменты в стиле "сосем с доверием" (как по мне эту звучит немного нелепо, в английском просто насколько я знаю "to suck at smth" вполне себе нейтральная фраза и я бы перевел ее как "совсем не уметь что-то"). Но как по мне это уже больше похоже на придирки.

toby3d commented:
Мигрировал репозиторий и поправил ссылку.

Minoru commented:
> Есть конечно моменты в стиле "сосем с доверием" […]. Но как по мне это уже больше похоже на придирки.

Именно такие «придирки» мне и интересны! Я хоть и не профессиональный переводчик, но мне все равно хочется, чтобы люди, читая мои переводы, не чувствовали, что это перевод. 

Смело создавай ишшуи и пулл-реквесты на любой чих; в процессе ревью разберёмся :)

Minoru commented:
@toby3d Комментарий обновить забыл!

Я почему на этом настаиваю: я про твой перевод узнал именно из того комментария. Просто клацнул по твоему нику, нашёл у тебя форк репозитория и создал в нём PR. Если кто-то сейчас захочет сделать то же самое, то он (или она) обломится, потому что у тебя в профиле репозитория больше нет. Просматривать организации, в которых ты состоишь, или листать твой activity stream возьмётся не каждый. Добавь в комментарий ссылку, упрости людям жизнь :)

toby3d commented:
@Minoru Обновил. Спасибо за уточнение.

balamosh commented:
Закончил редактирование /notes/index.html, но честно говоря не знаю как его загрузить через PR. А еще там появилось аж 2 русских перевода, один уже на офф. странице, кривоватый и со стандартным шрифтом, второй получше. Как будет время почекайте.

balamosh commented:
https://github.com/balamosh/trust/blob/gh-pages/notes/index.html
Вот тут файл, хз как по-другому это сделать.

toby3d commented:
@balamosh Просто создайте новую ветку со своими изменениями и создайте здесь PR указывющую на вашу ветку. Изменения будут сравнены и, если всё ок - я смержу ветки.

Minoru commented:
@balamosh У ГитХаба есть [пошаговое руководство по созданию пулл-реквестов](https://help.github.com/articles/creating-a-pull-request-from-a-fork/). Лучше переместить твои правки в отдельную ветку и делать пулл-реквест именно из неё — тогда ты сможешь делать другие изменения (и создавать новые пулл-реквесты!), не мешая ревью текущего. Но для простоты можно не заморачиваться.

На другие переводы я бегло глянул (первые пару экранов) — сыро. Но обязательно прочту полностью; авось оттуда какие-то идеи спереть можно :) Займусь этим во вторник.

Minoru commented:
@toby3d А этот перевод ты Ники ещё не отправлял? Мы работу по основному тексту закончили примерно тогда же, когда остальные два переводчика начали свою.

toby3d commented:
@Minoru Нет. Предлагаете сделать запрос?

Minoru commented:
А вот не знаю :)

С одной стороны, там сейчас в ишшуе замес на тему того, кто первый и чей перевод лучше. Если не вклиниться, можно потом остаться вообще не у дел.

С другой стороны, можно ещё немного потянуть и потом придти с самым отточенным переводом из имеющихся. Вот, работу @balamosh поревьювим-смержим, а завтра я почитаю чужие переводы и, возможно, утащу оттуда какие-то идеи.

В целом моя позиция такова: грустно будет, если труд окажется напрасным и в итоге игроки будут читать другой перевод.

Minoru commented:
В конце концов, можно набегать в багтрекеры других переводов и предлагать смержиться с нашим. Скорее всего, Ники не станет что-либо откатывть и смотреть на то, что ты первым объявил о начале своего перевода; поэтому нам придётся мержиться в перевод likzet.

toby3d commented:
Мда, неудобная ситуация... Я всё-таки попытаюсь в течении пары часов написать Ники письмо с запросом, а уже по ответу посмотрим что делать.

Minoru commented:
Пардон, сегодня чужое прочесть уже не успеваю, опять откладываю на завтра.

toby3d commented:
Это победа:
![image](https://user-images.githubusercontent.com/5412322/29179111-ebdf64a2-7e0c-11e7-8a8e-f0854b532486.png)
